pkg/extensions/sync/sync.go

```diff
@@ -510,6 +511,13 @@ func syncRegistry(ctx context.Context, regCfg RegistryConfig, upstreamURL string
 			// push from cache to repo
 			err = pushSyncedLocalImage(localRepo, tag, localCachePath, imageStore, log)
 
+			if errors.Is(err, zerr.ErrImageLintAnnotations) {
```

Comment:
no need to check this anymore, we have moved this check inside pushSyncedLocalImage()